ide/languages.go/nbproject/project.properties

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+javac.source=1.8
+
+# As of 2nd Jan 2023, we use the Grammar from:
+# https://github.com/antlr/grammars-v4/tree/4794c0c5371ce6b0711a306256f8f7706394ec6d/golang
+#
+golang.grammar.version=4794c0c5371ce6b0711a306256f8f7706394ec6d
```

Comment:
This is not needed or is it? But it reminds me, that parsers have the tendency to create incompatible code when the grammar is modified. So either `go.lang` should be a friend-only export or we must be willing to drop compatibility in the future (i.e. raise major version).

Reply:
I know it's a bit controversial topic nowadays, but I'd rather raise the release version on the go.lang module if needed.

Though I removed this stuff from here.